lib/internal/inspector_security.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const dns = require('dns');
+const util = require('util');
+
+const lookup = util.promisify(dns.lookup);
+
+const IP_RANGES = {
+  local: 'LOCAL',
+  private: 'PRIVATE',
+  public: 'PUBLIC'
+};
+
+function isValidIpV4(parts) {
```

Comment:
We already have this kind of functionality in core, it would be better to just reuse that.

Reply:
Do you mind pointing out where it is?

Reply:
@slonka I believe you can find this kind of validation on [internal/net.js](https://github.com/nodejs/node/blob/master/lib/internal/net.js)
https://github.com/nodejs/node/blob/2f1c356d7abe4949b5ee68e0724ed7e493fc03e1/lib/internal/net.js#L25-L37